sdk/eventhub/azure-eventhubs/azure/eventhub/eventprocessor/event_processor.py

```diff
@@ -0,0 +1,170 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# -----------------------------------------------------------------------------------
+
+from typing import Callable, List
+import uuid
+import asyncio
+import logging
+
+from azure.eventhub import EventPosition, EventHubError
+from azure.eventhub.aio import EventHubClient
+from .checkpoint_manager import CheckpointManager
+from .partition_manager import PartitionManager
+from .partition_processor import PartitionProcessor, CloseReason
+from .utils import get_running_loop
+
+log = logging.getLogger(__name__)
+
+
+class EventProcessor(object):
+    def __init__(self, eventhub_client: EventHubClient, consumer_group_name: str,
+                 partition_processor_factory: Callable[..., PartitionProcessor],
```

Comment:
The type annotation for partition_processor_factory does not provide the signature/arguments to the callable. And the docstring :param: annotation is using the old name.

Reply:
Added Callable signature and corrected :param: